client/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+
+# Create the executable target
+add_executable(client 
+    client.cpp
+    client_utils.cpp
+    client_utils.h
+    ../host/redis.cpp 
+    ../crypto/encryption_engine.cpp
+    ../gen-cpp/RPC.h
```

Comment:
In the Host CMakeLists.txt, we do `${CMAKE_BINARY_DIR}/gen-cpp/...`. Here we use `../`. I think this was left from when I was fiddling with somethings lol but we should stay consistent and use one or the other